build.cake

```diff
@@ -197,6 +197,26 @@ Task("TestAtlasConnectivity")
     );
 });
 
+Task("TestOcsp")
+    .IsDependentOn("Build")
+    .DoesForEach(
+        GetFiles("./**/MongoDB.Driver.Tests.csproj"),
+        testProject =>
+{
+    DotNetCoreTest(
+        testProject.FullPath,
+        new DotNetCoreTestSettings {
+            NoBuild = true,
+            NoRestore = true,
+            Configuration = configuration,
+
+            ArgumentCustomization = args => args
+                .Append("--filter FullyQualifiedName~OcspIntegrationTests")
```

Comment:
Interesting! I think that in the interest of speed, I would prefer to refrain from making this change since it currently works: we can certainly use your method if other test classes need to run for this test (although I suppose we could give them all a common prefix as well).

Also I'm not sure, but this https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test seems to indicate that the category filter is only supported by MSTest and not by xunit?

Reply:
I checked it in my PR and it worked.
We already have a common prefix for authentication https://github.com/DmitryLukyanov/mongo-csharp-driver/blob/efff2bedfe30262cbbfce51a5d6c620139b9ff1c/tests/MongoDB.Driver.Tests/Communication/Security/GssapiAuthenticationTests.cs#L26 but it's probably is not used anywhere. I will rely on your opinion here

Reply:
Resolving!